src/DIPS.Xamarin.UI/Controls/Sheet/SheetBehavior.cs

```diff
@@ -339,6 +428,31 @@ public void Hide()
             IsOpen = false;
         }
 
+        /// <inheritdoc />
+        public async Task BeforeRemoval()
+        {
+            if (m_modalityLayout == null) return;
+            if (m_sheetView == null) return;
+
+            var y = Alignment switch
+            {
+                AlignmentOptions.Bottom => m_modalityLayout.Height,
+                AlignmentOptions.Top => -m_modalityLayout.Height,
+                _ => throw new ArgumentOutOfRangeException()
+            };
+
+            await m_sheetView.SheetFrame.TranslateTo(m_sheetView.SheetFrame.X, y);

```

Comment:
m_sheetView.SheetFrame.X ? Shouldn't this be Y?

Reply:
No, first argument is `X` and second is `Y`. This is correct.